graphene/tests/issues/test_1264.py

```diff
@@ -0,0 +1,13 @@
+from graphene.types.enum import Enum
+
+
+def test_enum_iteration():
+    class TestEnum(Enum):
+        FIRST = 1
+        SECOND = 2
+
+    result = []
+    expected_values = ["FIRST", "SECOND"]
+    for c in TestEnum:
+        result.append(c.name)
+    assert result == expected_values
```

Comment:
This would be better-placed in `test_enum.py`, as this is not really a bug but rather a new "Feature" of the graphene-style enums.
Can you also add this test case for the instance-creation style of enums?
```python
def test_iterable_instance_creation_enum():
    TestEnum = Enum("TestEnum", [("FIRST", 1), ("SECOND", 2)])

    result = []
    expected_values = ["FIRST", "SECOND"]
    for c in TestEnum:
        result.append(c.name)
    assert result == expected_values
```